src/pages/index.js

```diff
@@ -67,6 +165,16 @@ addCardBtn.addEventListener("click", () => {
   addCardPopup.open();
 });
 
+avatarElement.addEventListener("click", () => {
+  avatarPopup.open();
```

Comment:
* You need to **`disable`** the submit button in the `card form` if the inputs are **empty**. It’s needed because I can click the button and add an empty card, but it should **not** happen. After the first submission, the button gets activated, and you need to **deactivate** it again. Even if I can't add an empty card, the button should look **disabled** (gray) anyway and have a `disabled` attribute. 


And the `avatar` submit button as well

```js

  formValidators["avatarForm"].resetValidation();

```